crates/ethereum/evm/src/lib.rs

```diff
@@ -93,6 +93,15 @@ impl<EvmFactory> EthEvmConfig<EvmFactory> {
         self.executor_factory.spec()
     }
 
+    /// Returns blob params by hard fork as specified in chain spec.
+    pub fn blob_max_and_target_count_by_hardfork(&self) -> [(SpecId, u64, u64); 2] {
```

Comment:
can we make this an iterator, then we don't have to change this later.

this should also clarify what the u64 tuple values represent.

returning tuples like this is never a good idea, but we're kinda forced to mirror revm here